### PR TITLE
changed get accommodations return type

### PR DESCRIPTION
**Pull request by hasmikharutyunyan**

### CODE REVIEW

File: HappyTravel.StaticDataMapper.Api/Services/Workers/AccommodationMapper.cs [hunk 1]
```diff
@@ -446,8 +447,9 @@ void Update(Contracts.MultilingualAccommodation accommodation, AccommodationKeyD
         }
 
 
-        private async Task<Dictionary<string, AccommodationKeyData>> GeCountryAccommodationBySupplier(
-            string countryCode, Suppliers supplier)
+        private async Task<List<( string SupplierCode, AccommodationKeyData AccommodationKeyData)>>
```
Comment (mingazhev) [inline HappyTravel.StaticDataMapper.Api/Services/Workers/AccommodationMapper.cs:450]: Redundant whitespace

File: HappyTravel.StaticDataMapper.Api/Services/Workers/AccommodationMapper.cs [hunk 2]
```diff
@@ -99,41 +99,42 @@ public async Task MapAccommodations(List<Suppliers> suppliers, CancellationToken
...
                     $"Started mapping of {supplier.ToString()} accommodations of country with code {country.Code}");
-                
+
                 var countryAccommodationsTree = await GetCountryAccommodationsTree(country.Code, supplier);
                 countryAccommodationsMappingSpan.AddEvent("Constructed country accommodations tree");
-                
-                
+
+
                 var countryAccommodationsOfSupplier =
```
Comment (mingazhev) [inline HappyTravel.StaticDataMapper.Api/Services/Workers/AccommodationMapper.cs:107]: Why not on the same line?

File: HappyTravel.StaticDataMapper.Api/Services/Workers/AccommodationMapper.cs [hunk 3]
```diff
@@ -99,41 +99,42 @@ public async Task MapAccommodations(List<Suppliers> suppliers, CancellationToken
...
                     await GeCountryAccommodationBySupplier(country.Code, supplier);
 
                 var notActiveCountryAccommodationsOfSupplier =
```
Comment (mingazhev) [inline HappyTravel.StaticDataMapper.Api/Services/Workers/AccommodationMapper.cs:110]: Same there

**Review by mingazhev — APPROVED** [review]
With comments

File: HappyTravel.StaticDataMapper.Api/Services/Workers/AccommodationMapper.cs [hunk 4]
```diff
@@ -244,22 +243,23 @@ void AddOrChangeActivity(Contracts.MultilingualAccommodation accommodation, bool
...
-                        Modified = utcDate
-                    };
-
-                    _context.Attach(accommodationToUpdate);
-                    _context.Entry(accommodationToUpdate).Property(ac => ac.IsActive).IsModified = true;
-                    _context.Entry(accommodationToUpdate).Property(ac => ac.Modified).IsModified = true;
-
-                    return;
-                }
+                // This situation is not real
```
Comment (mingazhev) [inline HappyTravel.StaticDataMapper.Api/Services/Workers/AccommodationMapper.cs:246]: Maybe remove this code or throw an exception in such case?

**Review by mingazhev — APPROVED** [review]

File: HappyTravel.StaticDataMapper.Api/Services/Workers/AccommodationMapper.cs (hunk 4 above)
Comment (hasmikharutyunyan) [reply HappyTravel.StaticDataMapper.Api/Services/Workers/AccommodationMapper.cs:246]: I leave this code, to remember in the future this situation (After testing I must do some refactorings)

**Pull request merged**